src/main/java/com/artipie/asto/s3/S3Storage.java

```diff
@@ -236,6 +235,61 @@ public S3Storage(final S3AsyncClient client, final String bucket) {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Uploads content using put request.
+     *
+     * @param key Object key.
+     * @param content Object content to be uploaded.
+     * @return Completion stage which is completed when response received from S3.
+     */
+    private CompletableFuture<Void> put(final Key key, final Content content) {
```

Comment:
@olegmoz let's rename this method to `upload` to be consistent.

Reply:
@Vatavuk sorry, but I think that current name is better. In S3 method for uploading file in single request is called `put`.

Reply:
@olegmoz ok